### PR TITLE
Renamed the whole package

### PR DESCRIPTION
**Pull request by gmagannaDevelop**
As per the **Name Change** section of the roadmap to [v1.0](https://github.com/edwinb-ai/Elysivm/projects/1).

Executed the test suite successfully, from the `REPL` as mentioned in [this issue](https://github.com/edwinb-ai/Elysivm/issues/34).

This should be good to go, I'm looking forward to reading your comments @edwinb-ai .

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 using Documenter
 
 # * Build the complete documentation
 makedocs(;
-    modules=[Elysivm],
+    modules=[LeastSquaresSVM],
     authors="Edwin Bedolla",
-    repo="https://github.com/edwinb-ai/Elysivm/blob/{commit}{path}#L{line}",
-    sitename="Elysivm",
+    repo="https://github.com/edwinb-ai/LeastSquaresSVM/blob/{commit}{path}#L{line}",
```
Comment (edwinb-ai) [inline docs/make.jl:8]: I'll be working on this, it seems that the documentation is not deploying correctly. It builds successfully, though.

**Review by edwinb-ai — COMMENTED** [review]
Fantastic work @gmagannaDevelop !!! I'll deal with the documentation, but this is great work! Thanks a lot.

File: docs/make.jl (hunk 1 above)
Comment (gmagannaDevelop) [reply docs/make.jl:8]: I'm glad to contribute ! :smile:

**Comment by gmagannaDevelop** [comment]
I'll review the roadmap to version 1.0 and the current issues to further contribute. I'm not sure if I have anything currently assigned but don't hesitate to notify me, I'd love to contribute more.

File: docs/make.jl (hunk 1 above)
Comment (edwinb-ai) [reply docs/make.jl:8]: Okay, I know what the problem is. The deployment keys do not resolve correctly due to the name change. I'll merge this and release a new hotfix of some sort.

**Pull request merged**